### PR TITLE
Fix run on sentence

### PR DESCRIPTION
**Pull request by lpww**
screen shot 2015-02-10 at 8 23 31 pm

Add grammar to a run on sentence in a bid to decrease confusion of readers. No other changes were made.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -63,11 +63,11 @@ connection, but historically has been over PS/2 or ADB connections.
 
 *In the case of Virtual Keyboard (as in touch screen devices):*
 
-- In modern capacitive touch screens when the user puts their finger on the
-  screen a tiny amount of current from the electrostatic field of the
-  conductive layer gets transferred to the finger completing the circuit
-  and creating a voltage dropping at that point on the screen so that the
-  ``screen controller`` raises an interrupt reporting the coordinate of
+- In modern capacitive touch screens; when the user puts their finger on the
```
Comment (kojiromike) [inline README.rst]: I don't think this is correct. The semicolon separates independent clauses, but

> In modern capacitive touch screens

isn't an independent clause. How about

> When the user puts their finger on a modern capacitive touch screen…

File: README.rst [hunk 2]
```diff
@@ -63,11 +63,8 @@ connection, but historically has been over PS/2 or ADB connections.
 
 *In the case of Virtual Keyboard (as in touch screen devices):*
 
-- In modern capacitive touch screens when the user puts their finger on the
-  screen a tiny amount of current from the electrostatic field of the
-  conductive layer gets transferred to the finger completing the circuit
-  and creating a voltage dropping at that point on the screen so that the
-  ``screen controller`` raises an interrupt reporting the coordinate of
+- When the user puts their finger on a modern capacitive touch screen, a tiny amount of current gets transferred to the finger. This completes the circuit through the electrostatic field of the conductive layer and creates a voltage drop at that point on the screen. The
```
Comment (tbodt) [inline README.rst]: This line is now too long, according to Travis.

**Comment by GortMcTabs** [comment]
@lpww created a PR on your fork to shorten split this up and maybe pass the build.

**Comment by BenjaminProgram** [comment]
I already did that in pull request #154

**Comment by GortMcTabs** [comment]
@BenjaminProgram That's alright too. I thought it would be better to have this whole modification on PR.

**Pull request closed without merging**